src/elements/checkbox/checkbox.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('semantic.ui.elements.checkbox', [])
+
+    .directive('smCheckbox', smCheckbox);
+
+  function smCheckbox() {
+    return {
+      restrict: 'E',
+      require: '?ngModel',
+      transclude: true,
+      replace: true,
+      template: '<div class="ui checkbox">' +
+                  '<input type="checkbox">' +
+                  '<label></label>' +
+                '</div>',
+
+      link: function(scope, element, attrs, ngModel, transclude) {
+
+        var checked = false;
+        var disabled = false;
+        var input = angular.element(element).find('input');
```

Comment:
`element` is already a jqLite or jQuery object, no need to `angular.element()` it